client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -908,6 +923,13 @@ static String endpoint(String[] indices, String endpoint, String[] suffixes) {
                 .addCommaSeparatedPathParts(suffixes).build();
     }
 
+    static String endpoint(String[] indices, String endpoint, String[] suffixes, String extraEndpoint, String[] extraSuffixes) {
```

Comment:
I think I'd make just call the `EndpointBuilder` directly rather than making a method. Unless you know that you are going to add a few more uses of the method.

Reply:
makes sense. thanks